config/sync/media.settings.yml

```diff
@@ -3,4 +3,4 @@ _core:
 icon_base_uri: 'public://media-icons/generic'
 iframe_domain: ''
 oembed_providers_url: 'https://oembed.com/providers.json'
-standalone_url: false
+standalone_url: true
```

Comment:
ISLE sets this; they shouldn't. That's a config variable that should be adjustable by the site owner. 
```
drupal/rootfs/etc/islandora/utilities.sh
549:    drush -l "${site_url}" -y config-set --input-format=yaml media.settings standalone_url true
```

One more that needs to be tweaked so that ISLE and Playbook can be in sync, sorry. I didn't realize because until now, we'd been using the standalone url.

Reply:
Updated in later commits; the ISLE thing does need to be addressed. https://github.com/Islandora-Devops/isle-dc/issues/326